deploy/crds/enterprise.splunk.com_clustermasters_crd.yaml

```diff
@@ -943,7 +1047,6 @@ spec:
                               x-kubernetes-int-or-string: true
                           required:
                           - port
-                          - protocol
```

Comment:
Is this removed because of the newer SDK?

Reply:
Yes, I assume so.  I didn't touch this config, just ran `make generate` and it threw out this diff.

Reply:
Having to re-add the protocol field is a limitation of the workaround to https://github.com/kubernetes/kubernetes/issues/91395 at the moment that we have to do anytime we generate CRDs.
 I have been looking into a better solution to automate this process or workaround the issue altogether, I can open a CSPL to formally track this.

Other projects that have hit the issue (kubernetes-sigs/controller-tools#444) mention upgrading their controller-gen version works around the issue, but I don't believe we use controller-gen to generate the crds. There are some other ideas mentioned here kubernetes/kubernetes#91395, but the automated ones seem to require modifying k8s api code.